app/controllers/v1/recipes_controller.rb

```diff
@@ -23,6 +25,11 @@ def show
     render json: recipe, serializer: Recipes::ShowSerializer
   end
 
+  def update
+    recipe = Recipe.find(params[:id])
+    render json: recipe, serializer: Recipes::ShowSerializer, status: 201
```

Comment:
Why are you sending the recipe back? Should we not send a success message instead?

Reply:
what happens if we try to update the ingredients of a recipe with more then 500 characters?

`validates_length_of :ingredients, maximum: 500`

Reply:
question 1: The Recipe has new attributes so I thought we might want to get it back. I'm sending a success code with the response.

Reply:
question 2: we are NOT testing for this now directly. But the current shoulda_matcher test does it for us.